defaults/main.yml

```diff
@@ -8,6 +8,7 @@ varnish_default_vcl_template_path: default.vcl.j2
 varnish_default_backend_host: "127.0.0.1"
 varnish_default_backend_port: "8080"
 
+varnish_listen_address: ""
```

Comment:
Both the Debian and RedHat tasks had "blank" or undefined for the value when configuring / launching varnish and that resulted in 0.0.0.0.  It might work with 0.0.0.0 but I just wanted to change as little as possible in case there was anything that wouldn't like that (and I'm not really able to test Debian easily).